CMakeLists.txt

```diff
@@ -50,6 +55,18 @@ execute_process(
 )
 set(VERSION_MD5 "0x${VERSION_MD5}")
 
+if (NOT DEFINED PPFW_FILENAME)
+	# Funny business here: trying to generate unique names to prevent renaming of shared test-drive builds by the browser on download to .ppfw-42.tar as it wont be recognized by the flasher
+	execute_process(
+		COMMAND date "+%y%m%d-%H%M"
+		OUTPUT_VARIABLE PPFW_FILENAME
+	)
+	string(STRIP ${PPFW_FILENAME} PPFW_FILENAME)
+	set(PPFW_FILENAME "portapack-mayhem_${PPFW_FILENAME}.ppfw.tar")
```

Comment:
Need to add oci?

Reply:
its intended use is to be shared via test-drive, so it wont really be one click ;)
When used in CI it'll just name it portapack-mayhem.ppfw.tar and the asset upload will rename it to oci

Reply:
It will still be from the website, (well 2).select the file from computer, click flash. And I think if we keep referring to them as oci it will end up making more sense to the end users. Just my 2c

Reply:
Meh.. imo its long enough already but whatever 🤷‍♂️

Should have just used oci instead of ppfw 😅